stdlib/public/core/SIMDVector.swift

```diff
@@ -848,7 +848,11 @@ extension SIMD where Scalar: FixedWidthInteger {
   /// Equivalent to `indices.reduce(into: 0) { $0 &+= self[$1] }`.
   @_alwaysEmitIntoClient
   public func wrappedSum() -> Scalar {
-    return indices.reduce(into: 0) { $0 &+= self[$1] }
```

Comment:
This wasn't being inlined, and doing the loop explicitly appears to fix that

Reply:
We should go ahead and land this change under a separate PR.